pkg/model/annotation.go

```diff
@@ -0,0 +1,46 @@
+package model
+
+import (
+	"errors"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+)
+
+var (
+	ErrAnnotationInvalidAppName   = errors.New("invalid app name")
+	ErrAnnotationInvalidTimestamp = errors.New("invalid timestamp")
+	ErrAnnotationInvalidContent   = errors.New("invalid content")
+)
```

Comment:
There are some custom error types that understood by httputils helper, e.g `ValidationError` – this will automatically instruct it to return HTTP code 400 with a properly formatted JSON body listing all the errors encountered